docs/_docs/monitoring-metrics/metrics.adoc

```diff
@@ -24,6 +24,23 @@ You can also access them programmatically.
 
 On this page, we've collected the most useful metrics and grouped them into various common categories based on the monitoring task.
 
+[NOTE]
+====
+By default, the JMX automatic configuration is disabled.
+To enable it, configure the following environment variables:
+
+* For `control.sh`, configure the `CONTROL_JVM_OPTS` variable
+* For `ignite.sh`, configure the `JVM_OPTS` variable
+
+For example:
+[source,shell]
+----
+JVM_OPTS="-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=${JMX_PORT} \
```

Comment:
Have a complete instruction for both Unix and Windows. For instance, "ignite.sh ... parameters"

Reply:
Not sure whether it's necessary to add separate tabs for the environment variable configuration (I'm pretty sure that it does not depend on the OS type).

Reply:
Not sure whether it's necessary to add separate tabs for the environment variable configuration.
As for the CONTROL_JVM_OPTS variable, its configuration depends on the Java version, so we can't provide a universal example here.